packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -125,6 +125,42 @@ export class IosSimulatorDevice extends DeviceBase {
         `${settings.location.latitude.toString()},${settings.location.longitude.toString()}`,
       ]);
     }
+    await exec("xcrun", [
+      "simctl",
+      "--set",
+      deviceSetLocation,
+      "spawn",
+      this.deviceUDID,
+      "notifyutil",
+      "-s",
+      "com.apple.BiometricKit.enrollmentChanged",
+      settings.biometricEnrollment ? "1" : "0",
+    ]);
+    await exec("xcrun", [
+      "simctl",
+      "--set",
+      deviceSetLocation,
+      "spawn",
+      this.deviceUDID,
+      "notifyutil",
+      "-p",
+      "com.apple.BiometricKit.enrollmentChanged",
+    ]);
```

Comment:
Why we do it twice and only once we send an argument?

Reply:
We don't do exactly the same thing, once we set the argument and the second one informs the system about the change.  https://keith.github.io/xcode-man-pages/notifyutil.1.html